src/elements/content-sidebar/activity-feed/task-form/messages.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @flow
+ * @file i18n messages
+ * @author Box
+ */
+
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
```

Comment:
since all of these have to do with the task form messages, I think it'd be a good choice to prefix all messages with the same name. Something like `be.tasks.taskForm`, then we could partition the details from that point.